addons/account/models/account_payment.py

```diff
@@ -247,72 +247,33 @@ def _get_valid_liquidity_accounts(self):
             self.journal_id.outbound_payment_method_line_ids.payment_account_id
         )
 
-    def _get_aml_default_display_map(self):
-        return {
-            ('outbound', 'customer'): _("Customer Reimbursement"),
-            ('inbound', 'customer'): _("Customer Payment"),
-            ('outbound', 'supplier'): _("Vendor Payment"),
-            ('inbound', 'supplier'): _("Vendor Reimbursement"),
-        }
-
     def _get_aml_default_display_name_list(self):
-        """ Hook allowing custom values when constructing the default label to set on the journal items.
-
-        :return: A list of terms to concatenate all together. E.g.
-            [
-                ('label', "Vendor Reimbursement"),
-                ('sep', ' '),
-                ('amount', "$ 1,555.00"),
-                ('sep', ' - '),
-                ('date', "05/14/2020"),
-            ]
-        """
         self.ensure_one()
-        display_map = self._get_aml_default_display_map()
-        values = [
-            ('label', _("Internal Transfer") if self.is_internal_transfer else display_map[(self.payment_type, self.partner_type)]),
-            ('sep', ' '),
-            ('amount', formatLang(self.env, self.amount, currency_obj=self.currency_id)),
-        ]
-        if self.partner_id:
-            values += [
-                ('sep', ' - '),
-                ('partner', self.partner_id.display_name),
+        payment_method = self.payment_method_line_id.name if self.payment_method_line_id else _("No Payment Method")
+        if self.ref:
+            return [
+                ('payment_method', payment_method),
+                ('sep', _(": ")),
+                ('memo', self.ref),
+            ]
+        else:
+            return [
+                ('payment_method', payment_method),
             ]
-        values += [
-            ('sep', ' - '),
-            ('date', format_date(self.env, fields.Date.to_string(self.date))),
-        ]
-        return values
 
     def _get_liquidity_aml_display_name_list(self):
```

Comment:
We can wait for the opinion of Quentin, i would say that if they want to override the function they can override _get_aml_default_display_name_list instead :smile:

Reply:
because the modification is done in master, it's totally fine to remove a method. In stable we need to avoid breaking the existing API, but not in master.

so, yeah, remove useless functions :)